Userland/Services/WebServer/Client.cpp

```diff
@@ -81,7 +81,7 @@ void Client::start()
             builder.append("\r\n"sv);
         }
 
-        auto request = builder.to_byte_buffer();
+        auto request = builder.try_to_byte_buffer().release_value_but_fixme_should_propagate_errors();
```

Comment:
If you're not propagating the error properly here, you can omit this change.

Reply:
Done!